x/id/types/did_document_test.go

```diff
@@ -259,3 +259,382 @@ func getBaseDocumentWithServices(services []Service) DidDocument {
 		Service: services,
 	}
 }
+
+func TestDidDocument_VerifyProof(t *testing.T) {
+
+	var testZone, _ = time.LoadLocation("UTC")
+	var testTime = time.Date(2016, 2, 8, 16, 2, 20, 0, testZone)
+	var testOwnerAddress, _ = sdk.AccAddressFromBech32("did:com:12p24st9asf394jv04e8sxrl9c384jjqwejv0gf")
+	okayDDOWrongSig := DidDocument{
+		Context: ContextDidV1,
+		ID:      testOwnerAddress,
+		Proof: &Proof{
+			Type:               KeyTypeSecp256k12019,
+			Created:            testTime,
+			ProofPurpose:       ProofPurposeAuthentication,
+			Controller:         testOwnerAddress.String(),
+			SignatureValue:     "uv9ZM4XusZl2q6Ei2O7aZW32pzwfg6ZQpBsQPb8cxzlFXWEyZLxem29fQBB4Py3W5gaXFEyPGruMXNsNDnr4sQ==",
+			VerificationMethod: "did:com:pub1addwnpepqwzc44ggn40xpwkfhcje9y7wdz6sunuv2uydxmqjrvcwff6npp2exy5dn6c",
+		},
+		PubKeys: PubKeys{
+			PubKey{
+				ID:         fmt.Sprintf("%s#keys-1", testOwnerAddress),
+				Type:       "RsaVerificationKey2018",
+				Controller: testOwnerAddress,
+				PublicKeyPem: `-----BEGIN PUBLIC KEY-----
```

Comment:
Since this content is reused all over the place, maybe we could save it in a file somehwere under a `testdata` directory?

Reply:
Agreed, I'll take care of it in the cleanup branch.